frontend/src/common/constants/stream/stream.ts

```diff
@@ -1,6 +1,7 @@
+import { ENV } from 'common/enums/enums';
 import { StreamPrivacy } from 'shared/build';
 
-export const STREAMING_SERVER_URL = 'rtmp://dev.streamlet.tk:1935/live';
+export const STREAMING_SERVER_URL = `rtmp://${ENV.PUSH_HOST}:1935/live`;
```

Comment:
Please move rtmp port also to .env variables and take into account that env variables in the frontend app add to the bundle while building docker image.

Probably better to rename PUSH_HOST to a more general (like SERVER_HOST ) name if it is used in several contexts